bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/DYBJets_HT-incl_Zpt-100to200/DYBJets_HT-incl_Zpt-100to200_run_card.dat

```diff
@@ -0,0 +1,271 @@
+#*********************************************************************
+# Beam polarization from -100 (left-handed) to 100 (right-handed)    *
+#*********************************************************************
+        0     = polbeam1 ! beam polarization for beam 1
+        0     = polbeam2 ! beam polarization for beam 2
+#*********************************************************************
+# PDF CHOICE: this automatically fixes also alpha_s and its evol.    *
+#*********************************************************************
+ lhapdf    = pdlabel     ! PDF set
+ $DEFAULT_PDF_SETS = lhaid
+ $DEFAULT_PDF_MEMBERS  = reweight_PDF
+#*********************************************************************
+# Renormalization and factorization scales                           *
+#*********************************************************************
+ F        = fixed_ren_scale  ! if .true. use fixed ren scale
+ F        = fixed_fac_scale  ! if .true. use fixed fac scale
+ 91.1880  = scale            ! fixed ren scale
+ 91.1880  = dsqrt_q2fact1    ! fixed fact scale for pdf1
+ 91.1880  = dsqrt_q2fact2    ! fixed fact scale for pdf2
+ 1        = scalefact        ! scale factor for event-by-event scales
+#*********************************************************************
+# Matching - Warning! ickkw > 1 is still beta
+#*********************************************************************
+ 1        = ickkw            ! 0 no matching, 1 MLM, 2 CKKW matching
+ 1        = highestmult      ! for ickkw=2, highest mult group
+ 1        = ktscheme         ! for ickkw=1, 1 Durham kT, 2 Pythia pTE
+ 1        = alpsfact         ! scale factor for QCD emission vx
+ F        = chcluster        ! cluster only according to channel diag
+ F        = pdfwgt           ! for ickkw=1, perform pdf reweighting
+ 5        = asrwgtflavor     ! highest quark flavor for a_s reweight
+ T        = clusinfo         ! include clustering tag in output
+ 3.0      = lhe_version       ! Change the way clustering information pass to shower.
+#*********************************************************************
+#**********************************************************
+#
+#**********************************************************
+# Automatic ptj and mjj cuts if xqcut > 0
+# (turn off for VBF and single top processes)
+#**********************************************************
+   T  = auto_ptj_mjj  ! Automatic setting of ptj and mjj
+#**********************************************************
+#
+#**********************************
+# BW cutoff (M+/-bwcutoff*Gamma)
+#**********************************
+  15  = bwcutoff      ! (M+/-bwcutoff*Gamma)
+#**********************************************************
+# Apply pt/E/eta/dr/mij cuts on decay products or not
+# (note that etmiss/ptll/ptheavy/ht/sorted cuts always apply)
+#**********************************************************
+   T  = cut_decays    ! Cut decay products
```

Comment:
supposed to be `False`, I guess? (to be synchronized with [inclusive sample](https://github.com/cms-sw/genproductions/blob/master/bin/GenValidation/updated_validation_cards/dy_mlm_mll50/dy01234mlm_inc/dyellell01234j_5f_LO_MLM_run_card.dat#L98))

Reply:
This won't have any effect to current setup where no mediate resonance exists.

1. If it's T=cut_decays with ptl=20
p p > z, z > e+ e- ==> electrons have pt > 20

2. If it's F=cut_decays with ptl=20
p p > z, z > e+ e- ==> electrons have no ptcut

3. If it's T/F=cut_decays with ptl=20
p p > e+ e- ==> electrons have pt > 20

Reply:
It was set to true by following the non-legacy cards
https://github.com/cms-sw/genproductions/blob/mg242legacy/bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/DYBJets_HT-incl_Zpt-100to200/DYBJets_HT-incl_Zpt-100to200_run_card.dat#L93

I am not that sure about it. Should I modify this?

Reply:
Probably not great to cut on the lepton PTs.  However, it shouldn't matter too much because the Zpt is at least 100 GeV.  I don't think I've ever seen any issue here in practice.  I'll check now though.

Reply:
@capalmer85 I didn't mean to say you should give cuts to lepton pts. ptl is just an **example**. What I meant to say is that "cut_decays" decide whether decay particles from mediate resonances are applied with cuts or not. And for your case, cut_decays = T won't really do anything since you haven't explicitly write the mediate resonance.

Reply:
I actually see plenty of events where the lepton pt is less than 20 GeV and the parent is the Z boson in the DYB 100-200 GeV sample.  So lepton cuts weren't there before and if the configuration hasn't changed, I guess I don't expect them there now.

Reply:
@panwarlsweet @capalmer85 @sihyunjeon Thanks for the swift responses. I was checking the run cards by 1-1 comparison with inclusive ones apart from processes so @sihyunjeon's comment should be right.

Reply:
hmmm... does look like there is some effect afterall.

https://capalmer.web.cern.ch/capalmer/DYB100to200_leptonpt.png

Reply:
@capalmer85 I think my comments have confused you. ptl is just an example. Not somthing that is asked to give to your gridpacks. I just wanted to explain to Sanghyun what cut_decays=T or F can or cannot do when (for instance) ptl=20 is given for "p p > z, z > e+ e-" or "p p > e+ e-" processes irrelevant to your cards.

Reply:
@capalmer85 Your cards are fine. Either you give cut_decays = T or cut_decays = F, what you get should stay the same.

Reply:
Resolved through Skype as I caused some confusions with the ptl comments. Resolving this.

Reply:
Thanks!  All good for me.